test/server.test.ts

```diff
@@ -1,17 +1,17 @@
-const request = require('supertest')
-const {createServer} = require('../src/server')
-const {logger} = require('../src/logger')
+import request from 'supertest'
+import { logger } from '../src/logger'
+import { createServer } from '../src/server'
 
-describe('server', function () {
-  let server
-  let webhook
+describe('server', () => {
+  let server: any
+  let webhook: any
 
   beforeEach(() => {
     webhook = jest.fn((req, res, next) => next())
-    server = createServer({webhook, logger})
+    server = createServer({ webhook, logger })
 
     // Error handler to avoid printing logs
-    server.use(function (err, req, res, next) {
+    server.use((err: any, req: any, res: any, next: any) => {
```

Comment:
Instead of `any`, could these use `Error`, `express.Request`, `express.Response`, and `express.NextFunction`